Week-4/assignment-2.js

```diff
@@ -0,0 +1,19 @@
+var Person = function () {};
+
+Person.prototype.initialize = function (name, age) {
+  this.name = name;
+  this.age = age;
+};
+
+var Teacher = function () {};
+
+Teacher.prototype.teach = function (subject) {
+  console.log(`${this.name} is teaching ${subject}`);
+};
+
+Object.setPrototypeOf(Teacher.prototype, Person.prototype);
```

Comment:
we can also simply assign prototypes..

Reply:
Used Objecct.setPrototypeOf() just to be make it explicit & clear and also as suggested by Sir in the video content...

Reply:
Yes got it.. just mentioning for your knowledge..(other way of doing it)